shacl12-vocabularies/shacl.ttl

```diff
@@ -120,6 +128,13 @@ sh:severity
 	rdfs:range sh:Severity ;
 	rdfs:isDefinedBy sh: .
 
+sh:shape
+	a rdf:Property ;
+	rdfs:label "shape"@en ;
+	rdfs:comment "Links from a node in the data graph to the shape(s) that it is expected to conform to."@en ;
```

Comment:
```suggestion
	rdfs:comment "Links from a node in the data graph to a shape that it is expected to conform to."@en ;
```

"The" somewhat implies a closed set of shapes.  This is not possible to restrict.

"(s)" somewhat hints a `rdf:List` could be provided.  I don't think that was the intention.

Going to "a shape" is more consistent with the multi-linking possibility that I think the spelling I changed away from meant.